xbmc/music/MusicArtUtils.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+ *      Copyright (C) 2018 Team XBMC
```

Comment:
Team KODI?

Reply:
Are we "Team Kodi" (or KODI)  in just new things now, replacing as we go, or going to replace globally in a PR as happened for the `http://Kodi.tv`?